modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/dag/schema.scala

```diff
@@ -0,0 +1,95 @@
+package org.tessellation.rosetta.server.model.dag
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+import enumeratum.{Enum, EnumEntry}
+import io.circe.Encoder
+import io.estatico.newtype.macros.newtype
+
+/** Represents schema changes specific to this project related to metadata or type Any fields which do not properly code generate
+  */
+object schema {
+
+  case class GenericMetadata()
+
+  @derive(eqv, show)
+  sealed trait ChainObjectStatus extends EnumEntry
+
+  object ChainObjectStatus extends Enum[ChainObjectStatus] with ChainObjectStatusCodecs {
+    case object Unknown extends ChainObjectStatus
+    case object Pending extends ChainObjectStatus
+    case object Accepted extends ChainObjectStatus
+
+    override def values: IndexedSeq[ChainObjectStatus] = findValues
+  }
+
+  trait ChainObjectStatusCodecs {
+    implicit val jsonEncoder: Encoder[ChainObjectStatus] =
+      Encoder.forProduct1("status")(_.toString)
+  }
+
+  @newtype
+  case class NetworkChainObjectStatus(value: ChainObjectStatus)
+
+  case class AccountBalanceResponseMetadata(sequenceNum: Int)
+
+  case class AccountCoinsResponseMetadata(sequenceNum: Int)
+
+  case class AccountIdentifierMetadata(alias: Option[String])
+
+  case class AmountMetadata()
+
+  case class BlockMetadata()
+
+  case class CallRequestParametersExampleEndpoint1(
+    paramOne: String
+  )
+
+  case class CallRequestParametersExampleEndpoint2(
+    paramOne: Int
+  )
+
+  case class CallRequestParameters(
+    callRequestParametersExampleEndpoint1: Option[CallRequestParametersExampleEndpoint1],
+    callRequestParametersExampleEndpoint2: Option[CallRequestParametersExampleEndpoint2]
+  )
```

Comment:
do you plan to use these classes somewhere? they are examples so imo they shouldn't be merged, as written in previous comment

Reply:
These can all be removed now, I believe they were used for some test but not required now?

Reply:
Removed.